src/main/java/org/folio/mr/domain/entity/FakePatronLink.java

```diff
@@ -0,0 +1,39 @@
+package org.folio.mr.domain.entity;
+
+import jakarta.persistence.Entity;
```

Comment:
We use different import order in our projects:
static imports
java.*
javax.*
org.*
com.*
all other imports